tools/tasks/css_task.ts

```diff
@@ -1,9 +1,11 @@
 import { Task } from './task';
+import Config from '../config';
 
 export abstract class CssTask extends Task {
 
+    // Is it still usefull if cache used ?
   shallRun(files: String[]) {
-    return files.some(f =>
+    return Config.ENABLE_SCSS || files.some(f =>
```

Comment:
Yes, but the piece of code is not properly working because of imports... Sometimes it appends that when I update some assets or sass files, I need to do a new `npm start` else the changes are not detected.